tests/ExampleApps/CoreExampleApp/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.6)
+if(${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} VERSION_GREATER "3.9")
+    cmake_policy(SET CMP0071 NEW)
+endif()
+if(APPLE)
+    cmake_policy(SET CMP0042 NEW)
+endif(APPLE)
+
+set(CMAKE_DISABLE_SOURCE_CHANGES ON)
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
+
+project(CoreExampleApp VERSION 1.0.0)
+
+if(NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE Release)
+endif()
+# Set default install location to installed-<Compiler_ID> folder in build dir we do not want to
+# install to /usr by default
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    set(CMAKE_INSTALL_PREFIX
+        "${CMAKE_CURRENT_BINARY_DIR}/installed-${CMAKE_CXX_COMPILER_ID}-${CMAKE_BUILD_TYPE}"
+        CACHE PATH "Install path prefix, prepended onto install directories." FORCE
+    )
+    message("Set install prefix to ${CMAKE_INSTALL_PREFIX}")
+endif()
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_DISABLE_SOURCE_CHANGES ON)
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
```

Comment:
Could we move all this duplicated code to some radium_function ?

Reply:
No. 

If you want that each demo has its own CMakeLists.txt, there will be lot of duplicated code. But this kind of configuration is application specific (here following the "recommandation" of Radium) and should not be set by the Radium package : 
 - cmake_policy are not intended to be permanent as they are set by some version of cmake to the wanted default value
 - Enabling in source build or changes, default build type  and default installation directory  should be left to the discretion of the application developper.